edition/voxel_tool.cpp

```diff
@@ -324,6 +324,39 @@ void VoxelTool::smooth_sphere(Vector3 sphere_center, float sphere_radius, int bl
 	}
 }
 
+void VoxelTool::do_surface(Vector3 sphere_center, float sphere_radius, float strength) {
+	ZN_PROFILE_SCOPE();
+	ZN_ASSERT_RETURN(sphere_radius >= 0.01f);
+	ZN_ASSERT_RETURN(strength >= 0.0f && strength <= 1.0f);
+
+	const Box3i voxel_box = Box3i::from_min_max(
+			math::floor_to_int(sphere_center - Vector3(sphere_radius, sphere_radius, sphere_radius)),
+			math::ceil_to_int(sphere_center + Vector3(sphere_radius, sphere_radius, sphere_radius)));
+
+	// TODO Perhaps should implement `copy` and `paste` with `VoxelBufferInternal` so Godot object wrappers wouldn't be
```

Comment:
This; but can be done later, I've done that for copy, just havent done yet for paste

Reply:
Removed the comment